packages/commerce-sdk-react/src/hooks/ShopperBaskets/utils.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2022, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+// NOTE: temporary file.. will be removed later
```

Comment:
@vmarta Why do we temporarily need to duplicate the `packages/commerce-sdk-react/src/hooks/utils.ts` file inside the `ShopperBaskets` folder?

Reply:
@adamraya it's because I wanted to narrow the scope of the changes to the Shopper Baskets only... for now.

Reply:
👍 Oh, I see we're in Draft mode not on final review.